label_studio/data_manager/actions/predictions_to_annotations.py

```diff
@@ -58,6 +58,13 @@ def predictions_to_annotations(project, queryset, **kwargs):
         )
         # Update counters for tasks and is_labeled. It should be a single operation as counters affect bulk is_labeled update
         project.update_tasks_counters_and_is_labeled(Task.objects.filter(id__in=tasks_ids))
+
+    try:
+        from stats.functions.stats import recalculate_stats_async_or_sync
+        recalculate_stats_async_or_sync(project, all=False)  # TODO I don't think we need all?
```

Comment:
```suggestion
        recalculate_stats_async_or_sync(project, all=False)
```

all=false is ok here